### PR TITLE
Release 1.11.0

### PR DESCRIPTION
**Pull request by soneillf5**
### Proposed changes
Update docs and configuration to use 1.11.0 version for images and references.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [ ] I have rebased my branch onto master
- [ ] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs-web/releases.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Releases
 
+### 1.11.0
+
+A list of changes can be found on Github at: [1.11 Release](https://github.com/nginxinc/kubernetes-ingress/releases/tag/v1.11.0)
```
Comment (pleshakov) [inline docs-web/releases.md:5]: Once we finalize the changelog, we'll need to copy it here.  the release date is needed as well.

we publish changelogs for all products at https://docs.nginx.com/

File: docs-web/integration-with-cis.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ The name of the resource and the labels in the selector must match the values yo
 
 Now the Ingress Controller pods are behind the IP configured in Step 2.
 
-If you deploy the [cafe example](https://github.com/nginxinc/kubernetes-ingress/tree/master/examples/complete-example), you will be able to send requests to the Ingress Controller pods using the following command:
+If you deploy the [cafe example](https://github.com/nginxinc/kubernetes-ingress/tree/v.1.11.0/examples/complete-example), you will be able to send requests to the Ingress Controller pods using the following command:
```
Comment (pleshakov) [inline docs-web/integration-with-cis.md:101]: could you remove this file for now (and also update https://github.com/nginxinc/kubernetes-ingress/blob/b50c771b813a9120a27bf1b861d6028fd9f7f0e4/docs-web/index.rst )? we will restore it once we officially publish this integration

File: docs/nginx-ingress-controllers.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ The table below summarizes the key difference between nginxinc/kubernetes-ingres
 | --- | --- | --- | --- |
 | **Fundamental** |
 | Authors | Kubernetes community | NGINX Inc and community |  NGINX Inc and community |
-| NGINX version | [Custom](https://github.com/kubernetes/ingress-nginx/tree/master/images/nginx) NGINX build that includes several third-party modules | NGINX official mainline [build](https://github.com/nginxinc/docker-nginx) | NGINX Plus |
```
Comment (pleshakov) [inline docs/nginx-ingress-controllers.md:17]: could you revert changes in this file? the links are for a different repo

**Review by pleshakov — COMMENTED** [review]
Hi @soneillf5 

There are a few missing updates:
* Makefile -  updating version there
* update deployments/helm-chart/Chart.yaml with the new versions and tags
* update Helm chart readme  with instructions and tags the for stable release — deployments/helm-chart/README.md
* missing doc pages updates 
    * installationbuilding-ingress-controller-image.md
    * installation/installation-with-helm.md
    * installation/installation-with-manifests.md
    * installation-with-operator.md
    
Also see https://github.com/nginxinc/kubernetes-ingress/commit/14b77a4252f261b2345902650ceb9c9c3d508082# and https://github.com/nginxinc/kubernetes-ingress/commit/22c65a823764680bc66c272a306f5bf7db5e0432#

I'll provide changelog feedback shorty via a different medium

File: docs-web/app-protect/configuration.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
 # Configuration
 This document describes how to configure the NGINX App Protect module
-> Check out the complete [NGINX Ingress Controller with App Protect example resources on GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/master/examples/appprotect).
```
[the remark this answers is not in the document]
Comment (soneillf5) [reply docs-web/app-protect/configuration.md:3]: fixed

File: docs-web/integration-with-cis.md (hunk 2 above)
Comment (soneillf5) [reply docs-web/integration-with-cis.md:101]: fixed

File: docs/nginx-ingress-controllers.md (hunk 3 above)
Comment (soneillf5) [reply docs/nginx-ingress-controllers.md:17]: fixed

File: tests/Makefile [hunk 5]
```diff
@@ -1,5 +1,5 @@
 CONTEXT =
-BUILD_IMAGE = nginx/nginx-ingress:edge
+BUILD_IMAGE = nginx/nginx-ingress:1.11.0
```
Comment (lucacome) [inline tests/Makefile:2]: I think the test files don't need to be changed, right?

File: deployments/helm-chart/Chart.yaml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 name: nginx-ingress
-version: 0.0.0-edge
-appVersion: edge
+version: 0.9.0
+appVersion: 1.11.0
 apiVersion: v1
 kubeVersion: ">= 1.16.0-0"
 description: NGINX Ingress Controller
```
Comment (pleshakov) [inline deployments/helm-chart/Chart.yaml:6]: `https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v1.11.0/deployments/helm-chart/chart-icon.png` for the icon?

File: deployments/helm-chart/README.md [hunk 7]
```diff
@@ -24,6 +24,7 @@ This step is required if you're installing the chart using its sources. Addition
 2. Change your working directory to /deployments/helm-chart:
     ```console
     $ cd kubernetes-ingress/deployments/helm-chart
+    $ git checkout v1.10.0
```
Comment (pleshakov) [inline deployments/helm-chart/README.md:27]:
```suggestion
    $ git checkout v1.11.0
```

File: tests/Makefile (hunk 5 above)
Comment (pleshakov) [reply tests/Makefile:2]: I don't see any difference. those test files are not used in pipelines and they don't affect the customer deliverable

**Review by pleshakov — APPROVED** [review]

**Review by ciarams87 — APPROVED** [review]

**Pull request merged**